app/net.go

```diff
@@ -39,6 +39,15 @@ func (app *Application) ServeHTTP(writer http.ResponseWriter, req *http.Request)
 		app.notConfiguredHandler.RequestHandle(ctx, writer, req)
 		return
 	}
+
+	var conn = app.conns.find(req.RemoteAddr)
+	if conn == nil { // tough
```

Comment:
Respond with `HTTP 500` or something similar here. Just returning from the handler is no good I guess. Also, `app.stats.responded()` should be called as well, since we've called `app.stats.requested()` at line 32. If not, the active connections counter in the status will go crazy.